engine/release-notes/index.md

```diff
@@ -13,896 +13,164 @@ redirect_from:
+- Added `RootlessKit`, `slirp4netns`, and `VPNKit` version reporting. [moby/moby#42330](https://github.com/moby/moby/pull/42330)
+- Added experimental support for SwarmKit cluster volumes (CSI). [moby/moby#41982](https://github.com/moby/moby/pull/41982)
+  - CLI: Added cluster volume (CSI) options to `docker volume`. [docker/cli#3606](https://github.com/docker/cli/pull/3606)
+  - CLI: Added cluster volume (CSI) support to `docker stack`. [docker/cli#3662](https://github.com/docker/cli/pull/3662)
+- Added support for SwarmKit jobs in `docker stack deploy`. [docker/cli#2907](https://github.com/docker/cli/pull/2907)
+- Added the `docker stack config` command to output the merged and interpolated config files as utilized by `stack deploy`. [docker/cli#3544](https://github.com/docker/cli/pull/3544)
+- Added a new `docker context show` command that prints the name of the current context. [docker/cli#3567](https://github.com/docker/cli/pull/3567)
+- Added a `--format=json` option to `docker inspect` and `docker ls`. [docker/cli#2936](https://github.com/docker/cli/pull/2936)
+- Added a `--quiet` option to `docker create` and `docker run` commands to suppress output when pulling an image. [docker/cli#3377](https://github.com/docker/cli/pull/3377)
+- Added a `--force` option to `docker network rm` subcommand. Causes CLI to return a 0 exit code even if the network does not exist. Has no effect on the server-side procedure for removing a network. [docker/cli#3547](https://github.com/docker/cli/pull/3547)
+- Added a `--signal` option to `docker stop` and `docker restart`. [docker/cli#3614](https://github.com/docker/cli/pull/3614)
+- Allow use of `STOPSIGNAL` via `docker commit`. [moby/moby#43369](https://github.com/moby/moby/pull/43369)
+- Added a new option to the `awslogs` log driver to allow skipping log stream creation in CloudWatch. [moby/moby#42132](https://github.com/moby/moby/pull/42132)
+- Added a new option to the `awslogs` log driver to specify the log format that is sent to CloudWatch. [moby/moby#42838](https://github.com/moby/moby/pull/42838)
+- Added a new option to the `fluentd` log driver to set the reconnection interval. [moby/moby#43100](https://github.com/moby/moby/pull/43100)
+- Added new options-setters to the Go API client: `WithTLSClientConfigFromEnv()`, `WithHostFromEnv()`, and `WithVersionFromEnv()`. [moby/moby#42224](https://github.com/moby/moby/pull/42224)
+- Added generation of shell command completion through `docker completion` subcommand. [docker/cli#3429](https://github.com/docker/cli/pull/3429)
+- API: Added a `Swarm` header to `GET /_ping` and `HEAD /_ping`, allowing single-request detection of Swarm support. [moby/moby#42064](https://github.com/moby/moby/pull/42064)
+- API: Added a `signal` parameter to `POST /containers/{id}/stop` and `POST /containers/{id}/restart` to set the signal used. [moby/moby#43206](https://github.com/moby/moby/pull/43206)
```

Comment:
We could consider adding back an API changes section, and link to https://docs.docker.com/engine/api/version-history/ "for a full overview of API changes in API v1.43 refer to .."

Reply:
ACK - I will bring up this suggestion with the docs team

Reply:
we recently changed the release note template so we're using the same template across all of the products. While it would perhaps be nice to add an API section for the engine section, it doesn't necessarily translate for all other products. So, let's avoid changing the heading now - we can revisit this later. Perhaps we could update the template to make it an optional heading for products where it applies.